Podfile

```diff
@@ -39,3 +40,15 @@ target 'ISICbalance' do
   end
 
 end
+
+target 'ISICbalanceExtension' do
+  # UI
+  pod 'SnapKit', '~> 4.0'
+  # Reactive
+  pod 'ReactiveSwift', '~> 4.0'
+  pod 'ACKReactiveExtensions', '~> 4.0'
+  pod 'ACKategories', '~> 6.0'
+  pod 'Alamofire', '~> 4.0'
+  pod 'SwiftSoup', '~> 1.7'
+  pod 'SwiftKeychainWrapper', '~> 3.2'
```

Comment:
pokud máš více targetů, které sdílejí stejné závislosti, můžeš si v Podfilu vytvořit globální funkci a v ní jednotlivé závislosti vypsat
```
def shared_pods
    pod 'SnapKit', '~> 5.0'
    pod 'ReactiveSwift', '~> 5.0'
end
```
potom pro každý target můžeš zavolat
```
target 'ISICbalanceExtension' do
    shared_pods
end